pkg/aws/policies.go

```diff
@@ -464,7 +464,7 @@ func (c *awsClient) AttachRolePolicy(roleName string, policyARN string) error {
 	return nil
 }
 
-func (c *awsClient) FindRoleARNs(roleType string, version string) ([]string, error) {
+func (c *awsClient) FindRoleARNs(roleType string, version string, onlyClassicPolicies bool) ([]string, error) {
```

Comment:
I am wondering if we can split this to two new funcs to remove the the need of passing the bool. 

```
func (c *awsClient) FindHCPRoleARNs(roleType string, version string) ([]string, error)
func (c *awsClient) FindClassicRoleARNs(roleType string, version string) ([]string, error)
```
This way we will be specific of what we are returning. We have too many places where we have a generic function and pass a series of bools to dictate what is returned. 

WDYT?

Reply:
Sounds good to me.
Let me add a new function `FindClassicRoleARNs`.
As for `FindHCPRoleARNs` we are still allowing to create HCP clusters with classic roles, I guess we need to check with BU and SRE, before filtering out classic roles for HCP in the CLI.